data/json/overmap/overmap_special/specials.json

```
@@ -4437,6 +4437,16 @@
     "occurrences": [ 0, 3 ],
     "flags": [ "CLASSIC", "WILDERNESS", "MAN_MADE" ]
   },
+  {
+    "type": "overmap_special",
+    "id": "drug_lab",
+    "overmaps": [ { "point": [ 0, 0, 0 ], "overmap": "drug_lab_north" } ],
+    "locations": [ "forest" ],
+    "city_distance": [ 20, 60 ],
+    "city_sizes": [ 8, -1 ],
+    "occurrences": [ 0, 2 ],
+    "flags": [ "CLASSIC", "WILDERNESS", "MAN_MADE" ]
```

Numbers pulled a bit out of my ass, but this should be rare.
```suggestion
    "occurrences": [ 1, 5 ],
    "flags": [ "UNQIUE", "CLASSIC", "WILDERNESS", "MAN_MADE" ]
```